tiled/client/container.py

```diff
@@ -653,7 +657,7 @@ def new(
             item["attributes"]["metadata"] = document.pop("metadata")
         # Ditto for structure
         if "structure" in document:
-            item["attributes"]["structure"] = STRUCTURE_TYPES[structure_family](
+            structure = STRUCTURE_TYPES[structure_family].from_json(
                 document.pop("structure")
             )
 
```

Comment:
Should we also check `(structure_family != StructureFamily.consolidated)` in line 670 below?

Reply:
Good catch. Actually, I think we should do the opposite: remove the special-case for `StructureFamily.container`. That was left from when `self._structure` for containers was sometimes `None`. This PR adds a `ContainerStructure` class, so now all client objects have a structure.